Modules/ITS/src/TrendingTaskITSFhr.cxx

```diff
@@ -0,0 +1,327 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+///
+/// \file    TrendingTaskITSFhr.cxx
+/// \author  Ivan Ravasenga on the structure from Piotr Konopka
+///
+
+#include "ITS/TrendingTaskITSFhr.h"
+#include "../../../Framework/src/RootClassFactory.h"
```

Comment:
I will move it to QualityControl/RootClassFactory.h, so you don't have to use such path.

Reply:
as agreed I left that path and you will modify it once you will move the file to QualityControl